server/schedulers/transfer_witness_leader.go

```diff
@@ -80,12 +80,7 @@ func (s *trasferWitnessLeaderScheduler) GetType() string {
 }
 
 func (s *trasferWitnessLeaderScheduler) IsScheduleAllowed(cluster schedule.Cluster) bool {
-	// TODO: make sure the restriction is reasonable
-	allowed := s.OpController.OperatorCount(operator.OpLeader) < cluster.GetOpts().GetLeaderScheduleLimit()
-	if !allowed {
-		operator.OperatorLimitCounter.WithLabelValues(s.GetType(), operator.OpLeader.String()).Inc()
-	}
-	return allowed
+	return true
```

Comment:
no any limit?

Reply:
Because when the witness becomes the leader, it cannot provide read and write requests, which will trigger the client to enter backoff to retry. If the transfer cannot be made in time, the client will time out. In addition, the strategy of witness raft log gc has been changed to ensure that there are enough logs and snapshots will not be generated during the transfer leader process.